client/themes/zenburn.css

```diff
@@ -253,3 +253,11 @@ body {
 #chat .toggle-content .body {
 	color: #d2d39b;
 }
+
+.btn-reconnect {
+	background: #e74c3c;
+	color: #fff;
+	border: 0;
+	border-radius: 0;
+	margin: 0;
```

Comment:
Repeating these overrides does not bother me: not all themes will have bordered/radiused buttons, and `.btn-reconnect` *is* an extension of `.btn`, so it makes sense. This reuses the Bootstrap's methodology (but not its values/design, I'm not adding any ties to Bootstrap here).